gateway_code/open_nodes/node_pycom.py

```diff
@@ -44,6 +48,22 @@
     b"machine.reset()\r\n",
 )
 
+PYCOM_UPDATE_BIN = "sudo /usr/bin/python3 /usr/local/share/pycom/eps32/tools/fw_updater/updater.py"
+PYCOM_FLASH_ERASE_HARD = "{bin} --pic -p {port} erase_fs"
+
+LOCAL_CONFIG_DIR = '/var/local/config'
```

Comment:
This won't be compatible with the multi ON feature that will come soon. Better get the path from the environment variable, as done here:

https://github.com/iot-lab/iot-lab-gateway/blob/f9acc3fcbcd5abf1048f8cb317903a52b95aaf5d/gateway_code/config.py#L41-L43

Reply:
Ok will improve this